scripts/deduplicate_schemas_safe.py

```
@@ -0,0 +1,153 @@
+        current[idx] = value
+    else:
+        current[last] = value
+
+def main():
+    spec = load_spec()
+    schemas = spec.setdefault("components", {}).setdefault("schemas", {})
+
+    # 1. Extract the full 401 error response schema
+    # Find the first 401 response and use it as the canonical version
+    error_401_schema = None
+    for path, methods in spec.get("paths", {}).items():
```

Ruff (0.15.15)

[warning] Loop control variable `path` not used within loop body

(B007)